_docs/_latest/setup/upgrade.md

```diff
@@ -151,6 +158,173 @@ change a certain resource, you'll need to create a new deployment of Forseti.
 
 Learn more about [Updating a Deployment](https://cloud.google.com/deployment-manager/docs/deployments/updating-deployments).
 
+{% capture upgrading_2_0_0_to_2_1_0 %}
+
+1. Open cloud shell when you are in the Forseti project on GCP.
+1. Checkout forseti with tag v2.1.0 by running the following commands:
+    1. If you already have the forseti-security folder under your cloud shell directory, 
+    run command `rm -rf forseti-security` to delete the folder.
+    1. Run command `git clone` to clone the forseti-security directory to cloud shell.
+    1. Run command `cd forseti-security` to navigate to the forseti-security directory.
+    1. Run command `git checkout tags/v2.1.0` to checkout version `v2.1.0` of Forseti Security.
+1. Download the latest copy of your Forseti server deployment template file from the Forseti server GCS 
+bucket (located under `forseti-server-xxxxxx/deployment_templates`).
+1. Update the `forseti-version` inside the deployment template to `v2.1.0`.
+1. Create file `forseti_server_v2_1_0.yaml` under `forseti-security/deployment-templates`.
+1. Copy and paste the content of the deployment template to file `forseti_server_v2_1_0.yaml`.
+1. Upload file `forseti_server_v2_1_0.yaml` back to the GCS bucket (`forseti-server-xxxxxx/deployment_templates`)
```

Comment:
Do we tell users what and how to update any changes in the `forseti_conf_server.yaml`?  Which in 2.1.0 is the cscc notifier.  Or other rule file changes?